src/tests/async/implement/implement.cs

```diff
@@ -24,7 +25,8 @@ public async Task<int> M1()
 
     class Derived1a : IBase1
     {
-        public async2 Task<int> M1()
+        [MethodImpl(MethodImplOptions.Async)]
```

Comment:
Are these necessary?

Reply:
I think these were a workaround for the change in the numerical value for the async methodimpl, that C# compiler was not updated for. 
I thought that has been fixed. Maybe this one was just missed when others were reverted.

Reply:
Yup, just missed during cleanup